extension-style/src/main/java/com/mapbox/maps/extension/style/sources/generated/VectorSource.kt

```diff
@@ -301,6 +321,16 @@ class VectorSource(builder: Builder) : Source(builder.sourceId) {
       properties[propertyValue.propertyName] = propertyValue
     }
 
+    /**
+     * A property to use as a feature id (for feature state). Either a property name, or
+     * an object of the form `{<sourceLayer>: <propertyName>}`. If specified as a string for a vector tile
+     * source, the same property is used across all its source layers.
+     */
+    fun promoteId(value: PromoteId) = apply {
+      val propertyValue = PropertyValue("promoteId", TypeUtils.wrapToValue(value.toValue()))
```

Comment:
Nit: do we still need wrapToValue since toValue() should have been Value type already?